src/orders/components/OrderDraftDetailsSummary/messages.ts

```diff
@@ -0,0 +1,36 @@
+import { defineMessages } from "react-intl";
+
+export default defineMessages({
```

Comment:
Use named exports over default ones:
```suggestion
export const messages = defineMessages({
```